ykman/_cli/openpgp.py

```
@@ -403,6 +403,29 @@ def set_touch(ctx, key, policy, admin_pin, force):
+    The value of touch cache time ranges from 0 to 255 seconds (0 means no cache).
+    """
+    session = ctx.obj["session"]
+
+    if admin_pin is None:
+        admin_pin = click_prompt("Enter Admin PIN", hide_input=True)
+
+    try:
+        session.verify_admin(admin_pin)
+        session.set_uif_cache_time(cache_time)
+        logger.info("Touch cache time set")
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)